recipes/opensees/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "OpenSees" %}
+{% set version = "3.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OpenSees/OpenSees/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fa805d2db131939b06042568f56b875c40d80a46c96b0d60dbfbfa0fbbceaaa6
+
+build:
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
+    - openmpi
+  host:
+    - python 3.11
+    - pip
+    - mumps
+    - eigen 3.4
```

Comment:
Please omit these pins so that the global pins are used.

Reply:
Removed version constraints, thank you for pointing that out